src/shortcut-manager.js

```diff
@@ -89,13 +89,8 @@ class ShortcutManager extends EventEmitter {
       return
     }
 
-    const shortcuts = _(cursor)
-      .map(this._parseShortcutDescriptor)
-      .flatten()
-      .compact()
-      .value()
-
-    return shortcuts
+    const shortcuts = _.map(cursor, this._parseShortcutDescriptor)
+    return _.values(_.compact(_.flatten(shortcuts)))
```

Comment:
@nickaversano Yeah, here you have to call `value` as the chain of functions are lazily evaluated. So I think we don't need `_.values` in this PR.